index.js

```diff
@@ -31,7 +36,13 @@ getFullNames(usersArray);
 // ***************************************************************************
 
 const getUsersCreditDetails = arr => {
-  // Your code goes here ...
+  let userDetails = [];
+  for (let user of arr){
+     let {firstName = user.firstName, lastName = user.lastName, balance = user.balance} = arr;
```

Comment:
this line doesnt work probably the way youre expecting.

it might be working because you are giving a default value in front of each value. arr is an array and you are trying to object destructure it.

ideally you could have just done 
```js
let { firstName, lastName, balance } = user
```

No need to call the arr value here